From 941d769292fbd3ecf82b1afe15b72745899d8a2b Mon Sep 17 00:00:00 2001
From: Steve Best <sbest@redhat.com>
Date: Wed, 13 Aug 2014 20:59:12 -0400
Subject: [x86] Mark Intel Broadwell processor as supported

Message-id: <20140813205911.26655.9761.sendpatchset@ibm-p7r2-01.lab.bos.redhat.com>
Patchwork-id: 87686
O-Subject: [PATCH RHEL7.1 BZ1089025] Mark Intel Broadwell processor as supported
Bugzilla: 1089025
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Milburn <dmilburn@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>

RHBZ#:
------
https://bugzilla.redhat.com/show_bug.cgi?id=1089025

Description:
------------

The Broadwell-U client processors CPU family 6 model 0x3D (61).

RHEL Version Found:
-------------------
RHEL 7.0

Brew:
-----
https://brewweb.devel.redhat.com/taskinfo?taskID=7840718

Upstream:
---------
This is a RHEL only change.

Test Status:
------------
Tested by me using intel-sharkbay-ult-03 which is CPU family 6 model 61.

before the patch
[    0.000000] Detected CPU family 6 model 61
[    0.000000] Warning: Intel CPU model - this hardware has not undergone testing by Red Hat and might not be certified.
Please consult https://hardware.redhat.com for certified hardware.

after the patch
the above warning msg is no longer written out.

---------------------------------------------------------------
Steve Best

Proposed Patch:
---------------

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/x86/kernel/setup.c b/arch/x86/kernel/setup.c
index 20ac806..79ec977 100644
--- a/arch/x86/kernel/setup.c
+++ b/arch/x86/kernel/setup.c
@@ -872,6 +872,7 @@ static void rh_check_supported(void)
   case 70: /* Crystal Well */
   case 63: /* Grantley/Haswell EP */
   case 62: /* Ivy Town */
+  case 61: /* Broadwell */
    break;
   default:
    if (boot_cpu_data.x86_model > 60) {
-- 
1.7.1